src/log_setup.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+from datetime import datetime
+
+
+def setup_logger():
+    logger = logging.getLogger("dataFetcherLogger")
```

Comment:
The logger name should be called something else given it's being used for things other than the data fetcher